README.md

```diff
@@ -135,6 +135,7 @@ To run as a desktop app:
 
    ```
    npm install electron
+   npm run install:electron
    node_modules/.bin/electron .
```

Comment:
I feel wrong putting `npm i electron` into an npm script, just feels really redundant, but if the majority agree then thats obviously the way to go